seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSourceReader.java

```diff
@@ -149,10 +158,13 @@ public void pollNext(Collector<SeaTunnelRow> output) throws Exception {
                                                                 ((CompatibleKafkaConnectDeserializationSchema)
                                                                                 deserializationSchema)
                                                                         .deserialize(
-                                                                                record, output);
+                                                                                record, output,
+                                                                                tablePath);
                                                             } else {
                                                                 deserializationSchema.deserialize(
-                                                                        record.value(), output);
+                                                                        record.value(),
+                                                                        output,
```

Comment:
wrapper `output` setting tablePath

Reply:
![image](https://github.com/apache/seatunnel/assets/76689593/6015ca05-4174-4bcc-9649-1361e9ba9b42)
Is this design intended? Please correct me if you have any questions

Reply:
Don't change the API, you can modify it inside the connector